lib/flight_configuration/base_dsl.rb

```diff
@@ -177,6 +220,106 @@ def transform(key, value)
       else
         value.send(transform)
       end
+    rescue
+      # NOTE: Ideally the error would be logged, however this can't be done
+      #       without forming a recursive loop
+      if active_errors?
+        config.errors.add(key.to_sym, type: :transform, message: 'failed to coerce the data type')
+      else
+        raise Error, "Failed to coerce attribute: #{key}"
+      end
+    end
+
+    def validate!(config)
```

Comment:
Similar to below regarding the `!`.

Reply:
0381df32a91097be7b0be25d332bbb221bd8d32f

Reply:
https://github.com/openflighthpc/flight_configuration/pull/1/commits/d2587478a5b1f2249c88a8110316397ab7d5611a